packages/geo/src/lib/search/search-selector/search-selector.component.html

```diff
@@ -24,5 +23,65 @@
         {{getSearchTypeTitle(searchType) | translate}}
       </mat-radio-button>
     </mat-radio-group>
+    <button
+      mat-menu-item
+      [matMenuTriggerFor]="setting">
+      <mat-icon svgIcon="settings"></mat-icon>{{ 'igo.geo.search.settings' | translate}}
+    </button>
   </mat-menu>
+
+  <mat-menu
+    #setting="matMenu"
+    class="no-border-radius">
+      <ng-container *ngFor="let source of getSearchSources()">
```

Comment:
Could this be a component on his own?

Reply:
Yes, in the future, a more elaborated menu in a "dialog" could be a component.   
That was my first track. 
This was a first draft and get something fast and easy to allow the user to choose some service parameter.

Reply:
I would love to see it in it's own component (a dialog or the same thing) but it's not my call..

Reply:
that make sense, a "search-settings" component for example.  
"search-selector" gonna be inside "search-settings" ?  
or configurable presence on the "search-bar" ?  
@mbarbeau welcome on board !